ocean_lib/common/aquarius/aquarius.py

```diff
@@ -195,7 +201,7 @@ def _parse_search_response(response):
         )
 
 
-def _parse_response(response, default_return):
+def _parse_response(response: object, default_return: Any) -> Any:
```

Comment:
The user can pass anything into default_return, so the returned value can be anything, I don't know how it can be tightened. The Response too, can be a mock, dict or Response object...